app/src/components/Map/LayerBlendingOptionsTooltip.jsx

```diff
@@ -46,6 +47,23 @@ class LayerOptionsTooltip extends Component {
       opacityRangeValue: this.opacityRangeConfig.value,
       hueRangeValue: this.hueRangeConfig.value
     };
+
+    this.closeTooltip = this.closeTooltip.bind(this);
+  }
+
+  shouldComponentUpdate(nextProps) {
+    if (nextProps.visible !== this.props.visible) {
+      if (nextProps.visible) window.addEventListener('click', this.closeTooltip);
+      else window.removeEventListener('click', this.closeTooltip);
```

Comment:
You need to add/remove it cause if you leave it the tooltip will open/close on any action due to the fact that the component is always mounted.